content/docs/components/serving/overview.md

```diff
@@ -4,62 +4,208 @@ description = "Model serving overview"
-|                | PyTorch        | [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/pytorch) | [x](https://www.kubeflow.org/docs/components/serving/pytorchserving/) |
-| Graph          | Transformers   | [x](https://github.com/kubeflow/kfserving/blob/master/docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb) | [x](https://docs.seldon.io/projects/seldon-core/en/latest/examples/transformer_spam_model.html)
-|                | Combiners       | Roadmap | [x](https://docs.seldon.io/projects/seldon-core/en/latest/examples/openvino_ensemble.html) |
-|                | Routers incl ([MAB](https://en.wikipedia.org/wiki/Multi-armed_bandit))         | Roadmap | [x](https://docs.seldon.io/projects/seldon-core/en/latest/analytics/routers.html) |
-| Analytics      | Explanations   | [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/explanation/alibi) | [x](https://docs.seldon.io/projects/seldon-core/en/latest/analytics/explainers.html) |
-| Scaling        | Knative        | [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/autoscaling) | |
-|                | GPU AutoScaling| [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/autoscaling) | |
-|                | HPA            |  x | [x](https://docs.seldon.io/projects/seldon-core/en/latest/graph/autoscaling.html) |
-| Custom         |  Container     | [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/custom) | [x](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html) |
-|                | Language Wrappers | | [python](https://docs.seldon.io/projects/seldon-core/en/latest/python/index.html), [java](https://docs.seldon.io/projects/seldon-core/en/latest/java/README.html), [R](https://docs.seldon.io/projects/seldon-core/en/latest/R/README.html) |
-|                | Multi-Container | | [x](https://docs.seldon.io/projects/seldon-core/en/latest/graph/inference-graph.html) |
-| Rollout        | Canary         | [x](https://github.com/kubeflow/kfserving/tree/master/docs/samples/rollouts) | [x](https://docs.seldon.io/projects/seldon-core/en/latest/examples/istio_canary.html) |
-|                | Shadow          | | x |
-| istio          |                 | x | x |
+Kubeflow supports two model serving systems that allow multi-framework model 
+serving: *KFServing* and *Seldon Core*. Alternatively, you can use a
+standalone model serving system. This page gives an overview of the options, so
+that you can choose the framework that best supports your model serving 
+requirements.
 
+## Multi-framework serving with KFServing or Seldon Core
+
+KFServing and Seldon Core are both open source systems that allow 
+multi-framework model serving. The following table compares 
+KFServing and Seldon Core. A check mark (**&check;**) indicates that the system
+(KFServing or Seldon Core) supports the feature specified in that row.
+
+<div class="table-responsive">
```

Comment:
This isn't rendering correctly. Are you sure it's correct? Otherwise LGTM

Reply:
Thanks @ellis-bigelow. The table looks OK on my screen. What rendering problem is it causing on your side?
https://deploy-preview-1454--competent-brattain-de2d6d.netlify.com/docs/components/serving/overview/